From: Bjarni Ingi Gislason <bjarniig@simnet.is>
Date: Thu, 15 Jan 2026 14:49:36 -0800
Subject: Fix misc issues in XZ man page (Closes: #1094436)

 Input file is xz.1

Output from "mandoc -T lint  xz.1": (shortened list)

      1 input text line longer than 80 bytes: xz, unxz, xzcat, lzm...

-.-.

Output from "test-groff -mandoc -t -ww -z xz.1": (shortened list)

      5 trailing space in the line

-.-.

Output from "test-groff  -mandoc -t -K utf8 -rF0 -rHY=0 -rCHECKSTYLE=10 -ww -z ":

troff:<stdin>:759: warning: trailing space in the line
troff:<stdin>:847: warning: trailing space in the line
an.tmac:<stdin>:871: style: use of deprecated macro: .PD
an.tmac:<stdin>:874: style: use of deprecated macro: .PD
an.tmac:<stdin>:1433: style: use of deprecated macro: .PD
an.tmac:<stdin>:1436: style: use of deprecated macro: .PD
an.tmac:<stdin>:1802: style: use of deprecated macro: .PD
an.tmac:<stdin>:1817: style: use of deprecated macro: .PD
an.tmac:<stdin>:2165: style: use of deprecated macro: .PD
an.tmac:<stdin>:2197: style: use of deprecated macro: .PD
an.tmac:<stdin>:2202: style: use of deprecated macro: .PD
an.tmac:<stdin>:2223: style: use of deprecated macro: .PD
an.tmac:<stdin>:2228: style: use of deprecated macro: .PD
an.tmac:<stdin>:2250: style: use of deprecated macro: .PD
an.tmac:<stdin>:2261: style: use of deprecated macro: .PD
an.tmac:<stdin>:2293: style: use of deprecated macro: .PD
an.tmac:<stdin>:2298: style: use of deprecated macro: .PD
an.tmac:<stdin>:2320: style: use of deprecated macro: .PD
an.tmac:<stdin>:2325: style: use of deprecated macro: .PD
an.tmac:<stdin>:2348: style: use of deprecated macro: .PD
an.tmac:<stdin>:2356: style: use of deprecated macro: .PD
an.tmac:<stdin>:2378: style: use of deprecated macro: .PD
troff:<stdin>:2612: warning: trailing space in the line
troff:<stdin>:2637: warning: trailing space in the line
troff:<stdin>:3002: warning: trailing space in the line

-.-

Tables:

  Use format 'l' (L), not 'c', for the last header column
if it is wider than data in its column.


These are all issues inherited from upstream and patch should be forwarded.

Forwarded: no
---
 src/xz/xz.1 | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/xz/xz.1 b/src/xz/xz.1
index 6e37b42..2249f56 100644
--- a/src/xz/xz.1
+++ b/src/xz/xz.1
@@ -775,7 +775,7 @@ The following table summarises the features of the presets:
 .PP
 .TS
 tab(;);
-c c c c c
+c c c c l
 n n n n n.
 Preset;DictSize;CompCPU;CompMem;DecMem
 \-0;256 KiB;0;3 MiB;1 MiB
@@ -863,7 +863,7 @@ That way no two presets are identical.
 .PP
 .TS
 tab(;);
-c c c c c
+c c c c l
 n n n n n.
 Preset;DictSize;CompCPU;CompMem;DecMem
 \-0e;256 KiB;8;4 MiB;1 MiB
@@ -2688,7 +2688,7 @@ Dictionary size is roughly equal to the decompressor memory usage.
 .PP
 .TS
 tab(;);
-c c c
+c c l
 c n n.
 Level;xz;LZMA Utils
 \-0;256 KiB;N/A
@@ -2713,7 +2713,7 @@ make the difference even bigger:
 .PP
 .TS
 tab(;);
-c c c
+c c l
 c n n.
 Level;xz;LZMA Utils 4.32.x
 \-0;3 MiB;N/A
@@ -3078,7 +3078,7 @@ Here are the relevant parts collected from those two tables:
 .PP
 .TS
 tab(;);
-c c
+c l
 n n.
 Preset;CompCPU
 \-0;0
